graylog2-web-interface/checkProductionBuild.js

```diff
@@ -0,0 +1,188 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+const puppeteer = require('puppeteer');
+const express = require('express');
+const cors = require('cors');
+
+const VENDORMODULE = 'vendor-module.json';
+const BUILDMODULE = 'module.json';
+
+const HEADLESS_SHELL = 'test/bin/headless_shell';
+
+const isExecutable = (filename) => {
+  try {
+    // eslint-disable-next-line no-bitwise
```

Comment:
Can we include a comment here as to why we need to disable bitwise operator linting? 
Should this just be a rule we change instead of disabling? 
Is there a way to code this that doesn't require disabling of linter?

Reply:
No I think this code needs to be written that way.
At least for me it is clear that way what is happening.
So I would go for the bit-wise OR since it is what is happening here.